### ISSUE TITLE
About GT data MPJPE and evaluation code.

### ISSUE THREAD

**Issue by MichelGao96** [bastianwandt/CanonPose#9]
Hello dear authors! Happy new year!
I have been going through your project lately, and I have also run your code. I have two questions:

1. How did you train your model with the ground truth 2D/3D Human3.6M dataset? I bet you don't need the skeleton morphing module with the GT data. But how did you calculate MPJPE with the GT data? I did the same things as you suggested in [issue 3](https://github.com/bastianwandt/CanonPose/issues/3), but my best result was 69.847mm MPJPE, far from the 65.9mm in your paper. Here are my steps for evaluation: center and normalize GT 2D data -> feed GT 2D data to the model to get canonical 3D prediction and rotation matrix (with "rodrigues")  -> rotation matrix matmul canonical 3D prediction to get 3D data in camera coordinates -> 3D prediction data and 3D GT data centered to get scale factor c with the help of [sirfoga](https://github.com/sirfoga/learnable-triangulation-pytorch/blob/master/mvn/utils/tred.py#L265) -> multiply the centered 3D prediction data with c and calculate the MPJPE with the centered 3D GT data. Can you figure out any mistake in my process?
2. Could you please release the code of model evaluation? I understand that you cannot share the original data, but it is totally fine as I already have the data. I just want to see your process.

I would appreciate it if you could answer my questions.

**Comment by jianlai123-123** [comment]
@MichelGao96 Hello, I'm a newcomer to learning attitude estimation, and I'm also very interested in this estimation code part. Could you share yours? thank you.
